_maps/database.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
You should probably ignore this whole chunk, I'm gonna make it pretty once I've actually finished the maps